### DISCUSSION TITLE
I installed Saleor in my own server and when I try to get the products only can I do it with a Token

### DISCUSSION THREAD

**Discussion by krasho** [saleor/saleor#14458] [Deployment]
I installed Saleor in my own server
https://saleor-api-staging.cookies.co/

and I installed in my computer the storefront to do testings:
https://github.com/saleor/storefront

My problem is I can't get the products without token, but if I change the API URL to https://demo.saleor.io/graphql/ I see the products correctly

how can I do the same, I will have customers without login and I would like to show the products.

please advise me

regards

**Comment by timuric** [comment]
Hi @krasho your products and variants are enabled in the channels you are querying 
<img width="1092" alt="image" src="https://github.com/saleor/saleor/assets/4006792/1123ea80-4385-4e92-8556-a427355b0fab">

**Comment by krasho** [comment]
Hello Timuric
Thanks my problem was the collection, it wasn't visible.
